ui/Screen/Funding/Pool/Outgoing/GetStarted.svelte

```diff
@@ -39,5 +42,5 @@
 <div class="wrapper">
   <Emoji emoji="💸" size="huge" />
   <p class="typo-text">Stream digital money to your favorite people.</p>
-  <Button on:click={getStarted}>Get started</Button>
+  <Button disabled={ongoingTx} on:click={getStarted}>Get started</Button>
```

Comment:
We could wrap this in a `<Tooltip>` explaining the user why it is disabled. Our tooltips already support conditionality, so that when the `value` prop is not present it'll just render the button without a tooltip.

Reply:
Wholeheartedly agree, as I have suggested that a few months back :b I will follow it up in a coming PR.

Reply:
https://github.com/orgs/radicle-dev/projects/14#card-54705424